fe/fe-core/src/main/java/org/apache/doris/planner/OlapScanNode.java

```diff
@@ -358,7 +358,12 @@ public void init(Analyzer analyzer) throws UserException {
     }
 
     public void init() throws UserException {
```

Comment:
Add a comment to explain `This function is only used for nereids `

Reply:
👌